src/PolygonWorldID.sol

```diff
@@ -51,7 +51,7 @@ contract PolygonWorldID is WorldIDBridge, FxBaseChildTunnel, Ownable2Step {
         WorldIDBridge(_treeDepth)
         FxBaseChildTunnel(_fxChild)
     {
-        receiveRootSelector = bytes4(keccak256("receiveRoot(uint256,uint128)"));
+        receiveRootSelector = bytes4(keccak256("receiveRoot(uint256)"));
         receiveRootHistoryExpirySelector = bytes4(keccak256("setRootHistoryExpiry(uint256)"));
```

Comment:
If these are only set in the constructor, can we make these immutable to save gas?

Reply:
yes